External logging configured in CM (#4252)
strimzi/strimzi-kafka-operator: documentation/api/io.strimzi.api.kafka.model.KafkaBridgeSpec.adoc

Use the `logging` property to configure loggers and logger levels.

You can set the log levels by specifying the logger and level directly (inline) or use a custom (external) ConfigMap.

> If a ConfigMap is used, you set `logging.valueFrom.configMapKeyRef.name` property to the name of the ConfigMap containing the external logging configuration.
> The `logging.valueFrom.configMapKeyRef.name` and `logging.valueFrom.configMapKeyRef.key` properties are mandatory.
> Default logging will be used if one of `name` or `key` is not set.

Reviewer: ```suggestion
If a ConfigMap is used, you set `logging.valueFrom.configMapKeyRef.name` property to the name of the ConfigMap containing the external logging configuration. Inside the ConfigMap, the logging configuration is described using `log4j2.properties`. The `logging.valueFrom.configMapKeyRef.name` and `logging.valueFrom.configMapKeyRef.key` properties are mandatory. Default logging is used if the `name` or `key` is not set.
```

Reviewer: Shall we write a line on what default logging means?
